app/views/categories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="categories">
+
+
+<div class="categories-list">
+  <% @categories.includes([:user]).order('created_at DESC').each do |category| %>
+    <% if can? :manage, category %>
+        <%= render category %>
+    <% end %>
+  <% end %>
+</div>
+
+<button class="newCatBtn">
+<%= link_to "New Category", new_category_path, class: 'newCatbtntext' %>
+</button>
+
```

Comment:
- [ ] The categories page also doesn't follow the correct design, this is how it looks in your page:

  ![image](https://user-images.githubusercontent.com/20567503/209024747-a7426cae-96ec-4e00-8423-b6505dfac7a8.png)

  And this is how it should look, the button should be fixed at the bottom and it should fill the whole width:

  ![image](https://user-images.githubusercontent.com/20567503/209024956-fee7fb23-4206-48d1-9255-0193f642d6b3.png)

  Kindly fix this as it is important to follow the design guidelines correctly.